src/rmodels.c

```diff
@@ -5664,7 +5664,7 @@ static Model LoadGLTF(const char *fileName)
                 }
 
                 // Load primitive indices data (if provided)
-                if (mesh->primitives[p].indices != NULL)
+                if (mesh->primitives[p].indices != NULL && mesh->primitives[p].indices->buffer_view != NULL)
```

Comment:
Please, could you put parenthesis around the individual conditions? I just reviewed it yesterday in many files...

Reply:
done